Makefile

```
@@ -190,6 +190,14 @@ submodules:
 	@git submodule sync
 	@git submodule update --init --recursive
 
+# Install CRDs into a cluster. This is for convenience.
+install: reviewable
+	kubectl apply -f cluster/charts/crossplane-types/crds/
```

```suggestion
install: $(KUBECTL) reviewable
	$(KUBECTL) apply -f cluster/charts/crossplane-types/crds/
```

I believe this change would allow folks to run the command even if they hadn't already installed `kubectl`